ui/src/ifql/ast/walker.ts

```diff
@@ -20,6 +44,19 @@ export default class Walker {
     return this.buildFuncNodes(this.walk(this.baseExpression))
   }
 
+  public get expressions(): FlatExpression[] {
+    const body = _.get(this.ast, 'body', []) as Body[]
```

Comment:
Does this break without the casting?

Reply:
The typing breaks because it doesn't know what the return value is from `_.get`